integration/simulation/simulation_docker_test.go

```diff
@@ -0,0 +1,117 @@
+package simulation
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
+)
+
+// TODO - Use individual Docker containers for the Obscuro nodes and Ethereum nodes.
+
+var (
+	enclaveDockerImg  = "obscuro_enclave"
+	nodeIDFlag        = "--nodeID"
+	enclaveDockerPort = "11000/tcp"
+)
+
+// This test creates a network of L2 nodes, then injects transactions, and finally checks the resulting output blockchain
+// The L2 nodes communicate with each other via sockets, and with their enclave servers via RPC.
+// All nodes live in the same process, the enclaves run in individual Docker containers, and the Ethereum nodes are mocked out.
+func TestDockerNodesMonteCarloSimulation(t *testing.T) {
```

Comment:
Maybe add a todo here that the plan is to start everything in docker as part of this test

Reply:
That's covered on line 17.